Posts/2021/02/XML-NewEmployees.md

```diff
@@ -0,0 +1,175 @@
+```yaml
+---
+post_title: 'Leveraging XML with New Employee PowerShell Scripts'
+username: s-r-turner
+categories: PowerShell
+tags: XML, ActiveDirectory
+featured_image:
+summary: using XML referencing to bolster up your new employee scripts with information.
+---
+```
+
+# Leveraging XML With New Employee PowerShell Scripts
+
```

Comment:
Remove the H1 title here. There should be no H1 headers. The title is automatically added to the post from the `post_title`  metadata.